.gitignore

```diff
@@ -0,0 +1,132 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This might only apply to me, but I use a Mac and typically have to also gitignore `.DS_Store` files. Hope this addition is fine!

```suggestion
# Custom
.DS_Store

# Byte-compiled / optimized / DLL files
```